test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetIVsApiTests/Utils.cs

```diff
@@ -47,7 +48,39 @@ public static SimpleTestPackageContext CreatePackage(string packageName, string
             return package;
         }
 
-        public static bool IsPackageInstalledInAssetsFile(NuGetConsoleTestExtension nuGetConsole, string projectPath, string packageName, string packageVersion)
+        public static void AssertPackageIsInstalled(NuGetApexTestService testService, ProjectTemplate projectTemplate, ProjectTestExtension project, string packageName, string packageVersion)
+        {
+            switch (projectTemplate)
+            {
+                case ProjectTemplate.ClassLibrary:
+                    testService.Verify.PackageIsInstalled(project.UniqueName, packageName, packageVersion);
+                    break;
+                case ProjectTemplate.NetCoreClassLib:
+                case ProjectTemplate.NetCoreConsoleApp:
+                case ProjectTemplate.NetStandardClassLib:
+                    var inAssetsFile = Utils.IsPackageInstalledInAssetsFile(project.FullPath, packageName, packageVersion);
+                    inAssetsFile.Should().BeTrue($"{packageName}-{packageVersion} should be installed in {project.Name}");
+                    break;
+            }
+        }
+
+        public static void AssertPackageIsNotInstalled(NuGetApexTestService testService, ProjectTemplate projectTemplate, ProjectTestExtension project, string packageName, string packageVersion)
+        {
+            switch (projectTemplate)
+            {
+                case ProjectTemplate.ClassLibrary:
```

Comment:
Same as above. 

Even during restore and build from the commandline, the way we tell apart if it's PPC or PR is by looking for tells such as packages.config and assets file. 

This won't last long term basically (I'd assume we'd want to Apex tests, next time we're trying to test a Legacy PR scenario.